.eslintrc.js

```diff
@@ -1,297 +1,297 @@
 module.exports = {
```

Comment:
why was this changed?

Reply:
This might have been the autoformatter. Honestly don't know

Reply:
Yes let's keep it as it was before the change